src/components/ingestion/FileIngestionEngine.js

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs'
+import AbstractIngestionEngine from '%src/components/ingestion/AbstractIngestionEngine'
+
+/**
+ * The FileIngestionEngine handles processing a video file. It is a concrete
+ * implementation of AbstractIngestionEngine, which reads in a file,
+ * converts it into an MPEG-TS stream represented as a series of Payloads,
```

Comment:
Chris I can't tell you what a relief it is to read "an MPEG-TS stream" without having to go through three review cycles. VERY MUCH CC @slifty

Reply:
You’ll love my secret refactor then.

Reply:
:ohno:

Reply:
Clearly we just pronounce MPEG very differently justin.

Reply:
I sing it like Hanson.